### DISCUSSION TITLE
WinterCMS/OctoberCMS Plugin Integration

### DISCUSSION THREAD

**Discussion by kndrckjvr** [rebing/graphql-laravel#905] [Q&A]
Has anyone tried implementing this package in Winter/October? I am having a hard time implementing this package as a Laravel Package.

**Comment by kndrckjvr** [comment]
so far I have tried using this https://luketowers.ca/blog/how-to-use-laravel-packages-in-october-cms-plugins/ guide to use this package, but having a hard time to know if I implemented it right

**Comment by Tony-S** [comment]
Hi kndrckjvr,
Did you get anywhere with this?
